servers/rendering/shader_compiler.cpp

```diff
@@ -381,7 +381,12 @@ void ShaderCompiler::_dump_function_deps(const SL::ShaderNode *p_node, const Str
 	}
 }
 
-static String _get_global_shader_uniform_from_type_and_index(const String &p_buffer, const String &p_index, ShaderLanguage::DataType p_type) {
+static String _get_global_shader_uniform_from_type_and_index(const String &p_buffer, const String &p_index, ShaderLanguage::DataType p_type, bool p_use_uint_as_index = false) {
+	String index_suffix = {};
```

Comment:
Strings are default initialized to `""` already.
```suggestion
	String index_suffix;
```

Reply:
Ah I know that, just a C++ habit of of always zero initializing a variable. Will fix it.

Reply:
Here initialisation like this would be wrong anyway at least style wise, if anything it should be either `String()` or `""`, good to get used to